### PR TITLE
simplify config interface

### PR DESCRIPTION
**Pull request by jspaaks**
(Illustrates an approach from #112)

### CODE REVIEW

File: howfairis/config.py [hunk 1]
```diff
@@ -17,9 +17,9 @@ class Config:
     """
 
     def __init__(self, repo: Repo, config_filename=None, ignore_remote_config=False):
-        self.default = Config._load_default_config()
-        self.repo = Config._load_repo_config(repo, ignore_remote_config)
-        self.user = Config._load_user_config(config_filename)
+        self._default = Config._load_default_config()
+        self._repo = Config._load_repo_config(repo, ignore_remote_config)
+        self._user = Config._load_user_config(config_filename)
```
Comment (jspaaks) [inline howfairis/config.py:22]:
```suggestion
        self._user = Config._load_user_config(config_filename)
        self._merged = self._merge_configurations()
```

File: howfairis/config.py [hunk 2]
```diff
@@ -101,15 +101,39 @@ def _load_user_config(config_filename):
         return user_config
 
     @property
```
Comment (jspaaks) [inline howfairis/config.py:104]:
```suggestion
```

File: howfairis/config.py [hunk 3]
```diff
@@ -100,16 +101,39 @@ def _load_user_config(config_filename):
                 "User configuration file should follow the schema.") from e
         return user_config
 
-    @property
-    def merged(self):
+    def _merged(self):
```
Comment (jspaaks) [inline howfairis/config.py:104]:
```suggestion
    def _merge_configurations(self):
```

**Review by sverhoeven — APPROVED** [review]
Looks good.

We will look at tests again in #146, now second assert failures are not detected.

**Comment by jspaaks** [comment]
Thanks for reviewing

**Pull request merged**